lib/modules/platform/azure/index.ts

```diff
@@ -461,6 +462,22 @@ async function getMergeStrategy(
   );
 }
 
+function mapMergeStrategy(
+  mergeStrategy?: MergeStrategy,
+): GitPullRequestMergeStrategy {
+  switch (mergeStrategy) {
+    case 'rebase':
+    case 'fast-forward':
```

Comment:
I'm not 100% sure about this mapping (fast-forward => rebase) but as the documentation explains, the only way to perform a fast-forward merge  on Azure DevOps is to rebase: https://learn.microsoft.com/en-us/azure/devops/repos/git/complete-pull-requests?view=azure-devops&tabs=browser#complete-a-pull-request

Reply:
cc @JamieMagee

Reply:
auto merge will only be done when branch is up to date, so git will simply push the commit to the target branch.